app/models/miq_worker/systemd_common.rb

```diff
@@ -69,7 +69,7 @@ def unit_file
           WorkingDirectory=#{working_directory}
           Environment=BUNDLER_GROUPS=#{bundler_groups.join(",")}
           ExecStart=/bin/bash -lc '#{exec_start}'
-          Restart=always
+          Restart=no
```

Comment:
👍   One process manager to rule them all... baby steps until we can stop micromanage things.